File: sdk/samples/all in on code/Visual Studio 2010/CSAzureTableStoragePaging/MvcWebRole/Models/MyModel.cs
/****************************** Module Header ******************************\
* Module Name:	TableStoragePagingUtility.cs
* Project:		AzureTableStoragePaging
* Copyright (c) Microsoft Corporation.
* 
* This is the MVC model. 
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Microsoft.WindowsAzure.StorageClient;
using Microsoft.WindowsAzure;

namespace MvcWebRole.Models
{
    public class CustomersSet
    {
        public List<Customer> Customers { get; set; }
        public bool ReadyToShowUI { get; set; }
    }

    public class Customer:TableServiceEntity
    {

        public string Name { get; set; }
        public int Age { get; set; }
       
        public Customer() 
        {
            this.PartitionKey = "part1";
            this.RowKey = Guid.NewGuid().ToString();
        }

    }

    public class CustomerDataContext : TableServiceContext
    {
        public CustomerDataContext(string baseAddress, StorageCredentials credentials) :
            base(baseAddress, credentials)
        {
           
        }

        public IQueryable<Customer> Customers
        {
            get
            {
                return CreateQuery<Customer>("Customers");
            }
        }
    }
}

File: sdk/samples/all in on code/Visual Studio 2010/VBAzureTableStoragePaging/MvcWebRole/Models/MyModel.vb
'***************************** Module Header ******************************\
'* Module Name:	MyModel.vb
'* Project:		AzureTableStoragePaging
'* Copyright (c) Microsoft Corporation.
'* 
'* This is the MVC model 
'* 
'* This source is subject to the Microsoft Public License.
'* See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
'* All other rights reserved.
'* 
'* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
'* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
'* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'\**************************************************************************

Imports Microsoft.WindowsAzure.StorageClient
Imports Microsoft.WindowsAzure
Namespace Models
    Public Class CustomersSet
        Public Property Customers() As List(Of Customer)

        Public ReadyToShowUI As Boolean
    End Class
    Public Class Customer
        Inherits TableServiceEntity

        Public Property Name() As String

        Public Property Age() As Integer

        Public Sub New()
            Me.PartitionKey = "part1"
            Me.RowKey = Guid.NewGuid().ToString()
        End Sub

    End Class
    Public Class CustomerDataContext
        Inherits TableServiceContext
        Public Sub New(ByVal baseAddress As String, ByVal credentials As StorageCredentials)
            MyBase.New(baseAddress, credentials)

        End Sub

        Public ReadOnly Property Customers() As IQueryable(Of Customer)
            Get
                Return CreateQuery(Of Customer)("Customers")
            End Get
        End Property
    End Class
End Namespace